package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "prettier": "^3.2.5"
+  }
+}
```

Comment:
Why?

Reply:
Oh, snap... this is not in the frontend directory.  This definitely doesn't belong here.  Please remove it.